qml/Chart.qml

```diff
@@ -97,8 +97,14 @@ Item {
 
         function draw(points, depth, initPost) {
             if (points) {
-                Util.update(serie, points, initPost, depth, minDepthToDraw, maxDepthToDraw, 1)
-                Util.update(serieInv, points, initPost, depth, minDepthToDraw, maxDepthToDraw, -1)
+                // If there is no user configuration, we set the max and min automatically
+                if(maxDepthToDraw == 0 && minDepthToDraw == 0) {
```

Comment:
I think this might be a little obscure place to put this logic. maybe we should TODO to refactor for readability, multiple sensors, etc.